### ISSUE TITLE
Could not find suitable constructor ...

### ISSUE THREAD

**Issue by brett-smith** [hypfvieh/dbus-java#160]
I'm seeing this log message come from dbus-java in one of my applications using the embedded daemon with the `master` branch . So this appears to only be server side, it's fine when I use my application with a native dbus broker.

```
14-02-2022 12:04:27 [DBus-Signal-Receiver-1] WARN  DBusSignal - Could not find suitable constructor for class org.freedesktop.dbus.interfaces.DBus$NameAcquired with argument-types: []
```

I don't think I am using the NameAcquired event anywhere, but is this anything to worry about?

**Comment by hypfvieh** [comment]
Thanks for the report. This is a bug related to changes I made recently.

The NameAcquired (and NameOwnerChanged) signal should always be emitted when a new client connected to the Bus and send 'Hello' message. 

The bug is, that the signal which should be send needs another type of constructor than NameAcquired provides. In the previous code the signal was created using new DBusSignal(...) instead of using NameAcquired. As far as I can see the usage of NameAcquired like introduced in the latest changes do not work because the signal is created differently than before.
Therefore using NameAcquired implementation only works for receiving the signal, not sending.

I reverted my changes causing the bug. I also fixed some issues in test setup (jnr-unixsocket transport was tested twice, native-transport was never used). I also bumped the version to 4.1.0.
